pkg/core/native/native_test/cryptolib_test.go

```diff
@@ -115,6 +115,26 @@ func TestCryptolib_TestBls12381Add_Compat(t *testing.T) {
 		hex.EncodeToString(arr[:]))
 }
 
+func TestKeccak256_Compat(t *testing.T) {
+	c := newCryptolibClient(t)
+
+	inputData := []byte("Keccak")
+
+	expectedHashHex := "868c016b666c7d3698636ee1bd023f3f065621514ab61bf26f062c175fdbe7f2"
+
+	script := io.NewBufBinWriter()
+	emit.AppCall(script.BinWriter, c.Hash, "keccak256", callflag.All, inputData)
+	stack, err := c.TestInvokeScript(t, script.Bytes(), c.Signers)
+
+	require.NoError(t, err)
+	require.Equal(t, 1, stack.Len())
+	itm := stack.Pop().Item()
+	require.Equal(t, stackitem.ByteArrayT, itm.Type())
+	actualHash := itm.Value().([]byte)
+	actualHashHex := hex.EncodeToString(actualHash)
```

Comment:
`actual := hex.EncodeToString(itm.Value().([]byte))`

You don't perform any other actions on these vars anyway.